modules/hardware/starfive/default.nix

```diff
@@ -0,0 +1,50 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib)
+    mkIf
+    mkMerge
+    mkOption
+    types
+  ;
+  inherit (config.hardware) mmcBootIndex;
+  cfg = config.hardware.socs;
+  starFiveSOCs = [ "starfive-jh7100" ];
+  anyStarFive = lib.any (soc: config.hardware.socs.${soc}.enable) starFiveSOCs;
+  isPhoneUX = config.Tow-Boot.phone-ux.enable;
```

Comment:
Until there are mobile platforms with starfive SoCs, we can drop any `phone-ux` configs.

Reply:
Ahh okay.